templates/sshd.template.yml

```diff
@@ -1 +1,4 @@
-# This file is deprecated; you can remove it from your app.yml
+run:
+  - exec: echo "The sshd.template.yml file is deprecated; you can remove it from your containers/app.yml file."
+  # TODO(2020-09-01): change to "exit 1"
+  - exec: bash -c "exit 0"
```

Comment:
do a date compare instead of a TODO

Reply:
Also, if it fails the rebuild it only does it at the very end instead of right up-front as with a missing template:

```
root@discourse-docker:/var/discourse# ./launcher rebuild app
Ensuring launcher is up to date
Fetching origin
remote: Enumerating objects: 4, done.
remote: Counting objects: 100% (4/4), done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 4 (delta 0), reused 2 (delta 0), pack-reused 0
Unpacking objects: 100% (4/4), done.
From https://github.com/discourse/discourse_docker
 + a2f3043...ef4f4cc fix-sshd   -> origin/fix-sshd  (forced update)
Launcher is up-to-date
Stopping old container
+ /usr/bin/docker stop -t 10 app
app
cat: templates/sshd.template.yml: No such file or directory
cd /pups && git pull && /pups/bin/pups --stdin
/usr/bin/docker: invalid reference format: repository name must be lowercase.
See '/usr/bin/docker run --help'.
cat: cids/app_bootstrap.cid: No such file or directory
"docker rm" requires at least 1 argument.
See 'docker rm --help'.

Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]

Remove one or more containers
rm: cannot remove 'cids/app_bootstrap.cid': No such file or directory
** FAILED TO BOOTSTRAP ** please scroll up and look for earlier error messages, there may be more than one.
./discourse-doctor may help diagnose the problem.
```

OTOH, who cares if there's an empty template there? Only those people *expecting* SSH to still work.